backend/app/Http/Controllers/UserController.php

```diff
@@ -59,11 +59,17 @@ public function show($id) {
     public function update(Request $request, $id) {
         $u = User::findOrFail($id);
         $attrs = $this->validateUser($request, $u);
+
+        if ($attrs['avatar'] ?? false) {
+            $attrs['avatar'] = $request->file('avatar')->store('avatars');
+        }
```

Comment:
If you are checking the avatar if it is not empty you can use this one below, this is related to truthy and falsy values
```suggestion
        if ($attrs['avatar']) {
            $attrs['avatar'] = $request->file('avatar')->store('avatars');
        }
```